xpsi/PostProcessing/_residual.py

```diff
@@ -293,7 +372,7 @@ def _add_data(self):
 
         data = self._ax_data.pcolormesh(self._signal.data.phases,
                                         channel_edges,
-                                        self._signal.data.counts/2.0,
```

Comment:
Since not dividing with 2 anymore, should the title in the plots change from `counts` to `counts/cycles`? This change means also that the old X-PSI signal plots will not look the same anymore if produced using the same script. Maybe that is at least worth to mention in the tutorial or in the Changelog?

Reply:
Agreed, 'counts/cycle' (well actually it's counts per bin per cycle I guess)?  and agreed we should mention this explicitly in the Changelog

Reply:
Yeah, I guess it is maybe counts / (bin / cycles)... if you count all the bins within all the shown cycles. But I am not sure what is best/correct. Would be just good to be clear that this has changed.